src/test/java/at/codersbay/datastructures/FractionalTest1.java

```diff
@@ -7,63 +7,69 @@
 
 public class FractionalTest1 {
 
-    @Test
-    // A fractional can be created if positive natural numbers are passed as numerator and denominator
-    public void testFractionalCreation() {
-        Fractional fractional = new Fractional(1, 2);
+	@Test
+	// A fractional can be created if positive natural numbers are passed as
+	// numerator and denominator
+	public void testFractionalCreation() {
+		Fractional fractional = new Fractional(1, 2);
 
-        assertThat(fractional.getNumerator()).isEqualTo(1);
-        assertThat(fractional.getDenominator()).isEqualTo(2);
-    }
+		assertThat(fractional.getNumerator()).isEqualTo(1);
+		assertThat(fractional.getDenominator()).isEqualTo(2);
+	}
 
-    @Test
-    // A fractional can be created if negative natural numbers are passed as numerator and denominator
-    public void testCreationWithNegativeNumerator() {
-        Fractional fractional = new Fractional(1, -2);
+	@Test
+	// A fractional can be created if a negative natural number is passed as
+	// denominator
+	public void testCreationWithNegativeDenominator() {
+		Fractional fractional = new Fractional(1, -2);
 
-        assertThat(fractional.getNumerator()).isEqualTo(1);
-        assertThat(fractional.getDenominator()).isEqualTo(-2);
-    }
+		assertThat(fractional.getNumerator()).isEqualTo(1);
+		assertThat(fractional.getDenominator()).isEqualTo(-2);
+	}
 
-    @Test
-    // An IllegalArgumentException should be thrown when a negative value is passed for the denominator
-    public void testCreationWithNegativeDenominator() {
-        Fractional fractional = new Fractional(-1, 2);
+	@Test
+	/*
+	 * Description was:
+	 * "An IllegalArgumentException should be thrown when a negative value is passed for the denominator"
+	 * But the actual test checks that negative value for the numerator + positive
+	 * for the denominator are valid. I've left the test in place and changed the
+	 * comment, because I found that this is more consistent with testCreationWithNegativeDenominator().
+	 */
+	// A fractional can be created if a negative natural number is passed as
```

Comment:
ah, of course, good catch - but this is a good example of where my comments on top of the test methods were actually more misleading than helpful - the method name was correct and the comment did not really add value but only confusion